### ISSUE TITLE
Matrix configuration status buttons broken

### ISSUE THREAD

**Issue by jhasse** [afonsof/jenkins-material-theme#20]
![matrix](https://cloud.githubusercontent.com/assets/80071/12653003/c63fda0c-c5ed-11e5-8b6e-6a99fb0c0c83.png)

**Comment by cobexer** [comment]
Relevant HTML:

```
	<table id="configuration-matrix">
		<tbody>
			<tr class="matrix-row">
				<td colspan="1" id="matrix-title">Configuration Matrix</td>
				<td title="browser" colspan="1" tooltip="browser" class="matrix-header">IE11</td>
			</tr>
			<tr>
				<td title="server" rowspan="1" tooltip="server" class="matrix-leftcolumn">el6</td>
				<td class="matrix-cell">
					<div>
						<a href="browser=IE11,server=el6/" class="model-link inside">
							<img title="Unstable " src="/static/c82d2d81/images/24x24/yellow.png" tooltip="Unstable " alt="Unstable" style="" height="24" width="24">
						</a>
					</div>
				</td>
			</tr>
		</tbody>
	</table>
```

**Comment by cobexer** [comment]
Just Discovered: in case the Matrix only has a single dimension it may also look like this:
```
	<div id="matrix">
		<h2>Configurations</h2>
		<a href="Server=el6/" class="model-link inside">
			<img title="Success " src="/static/c82d2d81/images/24x24/blue.png" tooltip="Success " alt="Success" style="" height="24" width="24">el6
		</a>&nbsp;
		<a href="Server=el7/" class="model-link inside">
			<img title="Success " src="/static/c82d2d81/images/24x24/blue.png" tooltip="Success " alt="Success" style="" height="24" width="24">el7
		</a>&nbsp;
		<img title="Not run" src="/static/c82d2d81/images/24x24/grey.png" tooltip="Not run" alt="Not run" height="24" width="24">&nbsp;
		<a href="Server=w2008r2/" class="model-link inside">
			<img title="Success " src="/static/c82d2d81/images/24x24/blue.png" tooltip="Success " alt="Success" style="" height="24" width="24">w2008r2
		</a>&nbsp;
		<a href="Server=w2012r2/" class="model-link inside">
			<img title="Success " src="/static/c82d2d81/images/24x24/blue.png" tooltip="Success " alt="Success" style="" height="24" width="24">w2012r2
		</a>&nbsp;
	</div>
```

**Comment by afonsof** [comment]
Fixed: Version 1.1.5

**Comment by cobexer** [comment]
Missed the second variant of the Matrix here, the second version has no `#configuration-matrix` div.
